### ISSUE TITLE
Weighted loss for classes

### ISSUE THREAD

**Issue by kaulanishasj** [tkuanlun350/Tensorflow-SegNet#8]
Hi, 

Thank you for sharing your code and helping us learn about segnet. 👍 . I had a small question.

In the model.py file (method: def cal_loss(logits, labels)), how is the array of  the weights generated?

Thank you.

**Comment by tkuanlun350** [comment]
Hi,
I use median-frequency balancing, basically use class frequency to penalize.
You can refer to [this paper](https://arxiv.org/abs/1411.4734) section 6.3.2 for detail.
